src/types/Theme.ts

```diff
@@ -52,6 +52,9 @@ export const textNoShadow: TextShadowParams = {
 export interface Theme {
   // The app scaling factor, which is the height of "normal" text:
   rem: (size: number) => number
+  units: {
+    rem: number
+  }
```

Comment:
Wait! No, no, no, we should not be using the `theme` inside worklets to begin with. This is because of the way the reanimated bridge works. Our theme is *huge*, and this enormous data structure would need to be be serialzed and sent cross-thread on every render.

So, if you need access to theme stuff inside a worklet, you really need to be doing something like this to begin with:

```ts
const exampleSize = theme.rem(3)

const someAnimation = () => {
  'worklet'
  return animationProgress * exampleSize
}
```

Now the worklet just captures a single number value, `exampleSize`, and not the whole theme. You can do this with colors or whatever else you need too.